libs/blocks/merch/merch.js

```diff
@@ -349,7 +380,15 @@ export async function initService(force = false) {
   const { env, commerce = {}, locale } = getConfig();
   commerce.priceLiteralsPromise = fetchLiterals(PRICE_LITERALS_URL);
   initService.promise = initService.promise ?? polyfills().then(async () => {
-    const commerceLib = await import('../../deps/commerce.js');
+    const { hostname, searchParams } = new URL(window.location.href);
+    let commerceLibPath = '../../deps/commerce.js';
+    if (/hlx\.(page|live)$|localhost$|www\.stage\.adobe\.com$/.test(hostname)) {
```

Comment:
![image](https://github.com/adobecom/milo/assets/330057/3746414b-b382-4024-b652-5a2baeb7e3df)

fyi: @narcis-radu

Reply:
Should we consider other STAGE environments like BACOM or Firefly ?

Reply:
Afaik BACOM doesn't have prices, and Firefly is not merch enabled yet.
This can be done later as part of merch enablement on Firefly.

Reply:
even with that, for now it's mostly for testing, and most of our pages are on cc/dc so i think we are good